From 89c47fd8b981ffa36cbc58a8a3039358958b4d05 Mon Sep 17 00:00:00 2001
From: Thiago Rocha <thiago_srocha01@hotmail.com>
Date: Thu, 8 Jun 2023 15:20:24 -0300
Subject: [PATCH] cherry-pick or format-patch

---
 src/components/modals/RegisterModal.tsx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/components/modals/RegisterModal.tsx b/src/components/modals/RegisterModal.tsx
index c9bae1e..2815dd6 100644
--- a/src/components/modals/RegisterModal.tsx
+++ b/src/components/modals/RegisterModal.tsx
@@ -74,7 +74,7 @@ export const RegisterModal = () => {
   );
 
   const footerContent = (
-    <div className="mt-3 flex flex-col gap-4 ">
+    <div className="mt-3 flex flex-col gap-4">
       <hr />
       <Button
         outline
-- 
2.40.1.windows.1

